src/shogun/mathematics/linalg/LinalgBackendBase.h

```diff
@@ -0,0 +1,61 @@
+#include <shogun/lib/config.h>
+#include <shogun/lib/common.h>
+#include <shogun/lib/SGVector.h>
+#include <shogun/io/SGIO.h>
+#include <shogun/mathematics/linalg/GPUMemoryBase.h>
+
+#include <iostream>
```

Comment:
we do have `SG_ERROR` or `SG_SERROR` or `SG_PRINT`... instead of `std::cerr`... just sayin'